talentprotocol/mvp
# Create Supporter Profile page

## Summary
Edit profile for supporters now with an About section
Supporters can create a display name, occupation, tags, website, socials and location
Supporter profile page now live both public and private


## Notion link
https://www.notion.so/talentprotocol/Supporter-Profile-2-2-562efec5abc843abb08935fa521e882f

## How to test?
Check if current edit profile for talents keeps working as before
Check if new edit profile for supporters is working correctly
As a supporter, try to open your own profile and check if everything is working correctly

## Notes
